src/app.js

```diff
@@ -3,18 +3,72 @@ const morgan = require('morgan')
 
 const router = require('./routes')
 
+const db = require('./db')
+
 const app = express()
 
 // Middleware
 app.use(express.json())
-if(process.env.NODE_ENV === 'development'){
+if (process.env.NODE_ENV === 'development') {
     app.use(morgan('dev'))
 }
 app.get('/', (req, res) => {
     res.send('Hello friend')
 })
 
+// CREATE USER ENDPOINT
+
+app.post('/users', async (req, res) => {
+    try {
+        const {
+            id,
+            firstName,
+            lastName,
+            email,
+            passwordHash,
+            gender,
+            jobRole,
+            department,
+            address,
+            createdAt,
+        } = req.body
+        const newUser = await db.query(
+            'INSERT INTO users ( id, "firstName" , "lastName" , email, "passwordHash" , gender , "jobRole" , department, address, "createdAt" ) VALUES ($1 , $2 , $3, $4, $5, $6, $7, $8, $9 , $10) RETURNING *',
+            [
+                id,
+                firstName,
+                lastName,
+                email,
+                passwordHash,
+                gender,
+                jobRole,
+                department,
+                address,
+                createdAt,
+            ]
+        )
+        res.json(newUser)
+    } catch (err) {
+        console.error(err.message)
+    }
+})
+
+//  CREATE POST ENDPOINT
+
+app.post('/article', async (req, res) => {
```

Comment:
i don't want to focus on structure too much for now till necessary but I think this should be handled in the `posts.js`.  

The code flow for now is a bit confusing for me so lets do that and see how it goes

Reply:
Done